nbs/index.ipynb

```diff
@@ -117,7 +114,13 @@
     "* Type `nano ~/.zshrc` to open the configuration of the terminal\n",
     "* Add the path to your mono installation: `export PKG_CONFIG_PATH=/usr/local/lib/pkgconfig:/usr/lib/pkgconfig:/Library/Frameworks/Mono.framework/Versions/6.12.0/lib/pkgconfig:$PKG_CONFIG_PATH`. Make sure that the Path matches to your version (Here 6.12.0)\n",
     "* Save everything and execute `. ~/.zshrc` \n",
-    "4. Navigate to the alphapept folder and install the package with `pip install .` (default users) or with `pip install -e .` to enable developers mode.\n",
+    "4. Install AlphaPept via pip: `pip install \"alphapept[gui-stable]\"`. If you want to use AlphaPept as a package withouth the GUI dependencies and strict version dependencies use `pip install alphapept`.\n",
```

Comment:
I have a stupid question here: what else can we use except for `xxx-stable`?

Reply:
You could use alphapept[gui] and then it would be w/o the version pinning.

Reply:
In an ideal case, we now have 5 extras:
* `stable`
* `gui`
* `gui-stable`
* `develop`
* `develop-stable`

Which can all be combined (although combing e.g. `gui-stable` and `gui` should work just fine, it is off course very redundant to do so)